src/ui/Quickview/Quickview.ts

```diff
@@ -347,7 +347,8 @@ export class Quickview extends Component {
         offset: {
           offset: '0,8'
         }
-      }
+      },
+      eventsEnabled: false
```

Comment:
Should we turn off gpu acceleration here too similar to below?

Reply:
It's not an issue here as now we only update the position when the user mouse overs the Quickview. It will almost never get fired.
I saw improvement in turning it off for the Facet's dropdown scroll/resize, which is fired a lot.